cmd/kubectl-testkube/commands/debug/create.go

```diff
@@ -0,0 +1,22 @@
+package debuginfo
```

Comment:
Can we make it just `tk create-ticket`?

Reply:
We can, I just did not want to saturate the list of commands more here. But on it